ui/src/app/lib/atlasmap-data-mapper/components/data-mapper-app.component.ts

```diff
@@ -33,7 +33,11 @@ import { Subscription } from 'rxjs';
   moduleId: module.id,
   encapsulation: ViewEncapsulation.None,
   templateUrl: './data-mapper-app.component.html',
-  styleUrls: ['data-mapper-app.component.css'],
+  styleUrls: ['data-mapper-base.component.css',
```

Comment:
@igarashitm @seanforyou23 
Ok - here's my quandry...
   Lets examine the toolbar component since it's fairly straightforward.
   I created:
   ui/src/app/lib/atlasmap-data-mapper/components/data-mapper-toolbar.component.css
   it includes base:
`@import 'data-mapper-base.component.css';`
   All of the data-mapper-toolbar component css entries are preceded by '.DataMapperUI' so as
   to not conflict with anybody who might include the data mapper into their app as a child 
   app/service (Syndesis).
` i.e.:  .DataMapperUI .toolbar { margin:1px 0px; padding:10px; }`
   The main atlasmap app (data-mapper-app.component.ts) does NOT include 
   data-mapper-toolbar.component.css in its styleUrls.  In theory the main atlasmap app should
   just include data-mapper-base.component.css.

   This fails...  (toolbar is all messed up).

   What's interesting is that if I remove the '.DataMapperUI' prefix -
` i.e.:  .toolbar { margin:1px 0px; padding:10px; }`
   then all is well.  The issue seems to be that the DataMapperUI class is defined in the
   data-mapper-app.component.html which is not known to the 
   data-mapper-toolbar.component.css (even though if you inspect the elements you can see
   the DataMapper UI class).

   I don't want to create an artifical dependency on every component back to the
   data-mapper-app.component just to satisfy a css issue.  The most straightforward solution 
   would be to just rename the top-level classes for each major css grouping and use that -
`i.e. .ADMtoolbar { margin:1px 0px; padding:10px; }`
   That would fix the issue and provide with a non-conflicting CSS name.

   WDYT??

Reply:
I think base CSS should not be referred from any template, but only be inherited by the actual sub CSS files. The base CSS only defines global variables, maybe the class name of `DataMapperUI` could be a first variable.
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties
Then we will migrate all duplicated values like `--global-bg-color` into base CSS, and let sub CSSs use that variable down the road.
It also sounds like we need one more CSS file, like data-mapper-app.css so standalone app template would refer but Syndesis usage doesn't.